Cython/Compiler/Symtab.py

```diff
@@ -269,6 +271,20 @@ def redeclared(self, pos):
     def all_alternatives(self):
         return [self] + self.overloaded_alternatives
 
+    def best_function_match(self, scope, arg_types, fail_if_empty=False, arg_is_lvalue=None):
```

Comment:
`arg_is_lvalue` reads to me like it should be a single boolean, rather than a list of booleans

Reply:
I changed the name to `arg_is_lvalue_array`.